serverless.yml

```diff
@@ -35,12 +35,12 @@ provider:
 
   environment:
     STAGE: ${self:provider.stage}
-    DEPLOY_STAGE: ${env:DEPLOY_STAGE}
-    TWILIO_MESSAGING_SERVICE_SID: ${env:TWILIO_MESSAGING_SERVICE_SID}
-    TWILIO_ACCOUNT_SID: ${env:TWILIO_ACCOUNT_SID}
-    TWILIO_AUTH_TOKEN: ${env:TWILIO_AUTH_TOKEN}
-    DB_CLUSTER_ARN: ${env:DB_CLUSTER_ARN}
-    DB_SECRET_ARN: ${env:DB_SECRET_ARN}
+    DEPLOY_STAGE: ${ssm:/stopcovid/${self:provider.stage}/deployStage}
+    TWILIO_MESSAGING_SERVICE_SID: ${ssm:/stopcovid/${self:provider.stage}/twilioMessagingServiceSID~true}
+    TWILIO_ACCOUNT_SID: ${ssm:/stopcovid/${self:provider.stage}/twilioAccountSID~true}
```

Comment:
anything with `~true` at the end is a secure string that needs to be decrypted.

See https://console.aws.amazon.com/systems-manager/parameters/?region=us-east-1 to check the parameters that are currently set.